_includes/specials/bech32/zh/02-stats.md

```diff
@@ -1,6 +1,6 @@
 正如[上周所述][bech32 easy]，实现 segwit 仅花费应当是容易的。然而我们怀疑一些管理者可能会想知道是否有足够多人使用 segwit 来证明他们的团队值得在此方面花费开发精力。本周，我们查看了追踪各种 segwit 采用统计数据的网站，以便你可以决定它是否足够流行，以至于你的钱包或服务如果不支持它会成为异类。
 
-Optech 在我们的[仪表盘][optech dashboard]上追踪 segwit 使用的统计数据；另一个追踪相关统计数据的网站是 [P2SH.info][]。我们看到平均每个区块大约有 200 个输出被发送到原生 segwit 地址 (bech32)。这些输出随后在大约 10% 的比特币交易中被花费。使得涉及原生 segwit 地址的支付比几乎所有的山寨币都更受欢迎。
+Optech 在我们的仪表盘上追踪 segwit 使用的统计数据；另一个追踪相关统计数据的网站是 [P2SH.info][]。我们看到平均每个区块大约有 200 个输出被发送到原生 segwit 地址 (bech32)。这些输出随后在大约 10% 的比特币交易中被花费。使得涉及原生 segwit 地址的支付比几乎所有的山寨币都更受欢迎。
```

Comment:
@hulatown would you mind taking a look at this change where I removed the link, but see the EN version in this PR and we can have something similar for ZH?

Reply:
Hi @bitschmidty Mike, sorry that I just notice this PR. And yes, the change is correct for ZH. 
Should I open another PR for this?

Reply:
I think it is good as is as long as the text makes sense. If it needs a change, feel free to open a new PR